src/test/java/MainTest.java

```diff
@@ -0,0 +1,15 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest {
```

Comment:
Do remember to add proper tests before the iP deadline!

Reply:
Thanks for reminding me!